ideagenerator/data.js

```diff
@@ -440,21 +452,23 @@ satirical
 
 #wildcard:
 with <a> [hand-drawn, watercolour, graphic-novel, claymation, retro, low-poly, dream-like, abstract, cardboard-cutout] art style
-controlled with [just one button, only two buttons, only the mouse, your voice]
-in which [you explode, time freezes, time slows down, time speeds up, you shrink, you get bigger, the game gets harder] whenever you [stop moving, jump]
+controlled [with, by] [just one button, only two buttons, only the mouse, your voice, a Wii remote, a joystick, twitch chat]
+in which [you explode, time freezes, time slows down, time speeds up, you shrink, you get bigger, the game gets harder] whenever you [stop moving, jump, die]
 in which you only have one [life, item, inventory-slot, chance]
-with <a> [time-travelling, grappling-hook] mechanic
+with <a> [time-travelling, grappling-hook, golf-swing, natural language processing, cheating] mechanic
 which teaches the player [moral lessons, mathematics, geography, science, history, programming, cooking, random trivia]
 with challenging boss fights
 with a [cooperative, competitive] splitscreen mode
 with an emphasis on [crafting, exploration, alchemy, creativity]
 in which you need to collect [dreams, colours, memories]
 with unlockable achievements
-with lots of [procedurally generated content, character customization options]
+with lots of [procedurally generated content, character customization options, memes, bugs, errors]
 with <a> [online,] leaderboard
 designed with speedrunning in mind
 with destructible terrain
 with a built-in level editor
 with a [funny, moving] story
+with livestream chat integration
+with the only sound coming from <a> [Microsoft Bob, Windows error tones, synthesizer, orchestra, hard drive wirring]
```

Comment:
```suggestion
only using sounds coming from <a> [Microsoft Bob, error tones, synthesizer, orchestra, hard drive whirring]
```
Not sure what Microsoft Bob is 🤔

Reply:
I misremembered. It's actually [Microsoft Sam](https://tetyys.com/SAPI4/)

Reply:
Hmmm, maybe it's a bit too specific then ? I guess it would be better to keep it generic & broad. I also think the `<a>` rule won't work for the following items entered as plurals (error tones). I might be wrong though, @SebLague does the `<a>` rule also handle plurals ?